.github/workflows/scriptcheck.yml

```diff
@@ -154,9 +154,21 @@ jobs:
         env:
           PYTHONPATH: ./addons
 
-      - name: test htmlreport
+      - name: test htmlreport (standalone)
         run: |
-          htmlreport/test_htmlreport.py
+          htmlreport/tests/test_htmlreport.py
+          cd htmlreport
+          ./check.sh
+
+      - name: test htmlreport (pip)
+        if: matrix.python-version != '3.5' && matrix.python-version != '3.6'
```

Comment:
Why are these versions excluded? Please add a small comment about it.

Reply:
That relates to the [`setuptools-scm`](https://github.com/danmar/cppcheck/pull/6335#discussion_r1585363401). Starting from the [v6.0.0](https://github.com/pypa/setuptools_scm/blob/main/CHANGELOG.md#v600) they have dropped support for EOL Python versions and required Python > 3.6.

Reply:
Comment was added.